Zukini.UI.Examples.Features/Steps/SmokeTestSteps.cs

```diff
@@ -81,5 +88,47 @@ public void ThenIShouldSeeThatTheTableTagIsSupportedForTheFollowing(Table table)
                 Assert.IsTrue(page.IsBrowserSupported(browserName), $"Expected browser {browserName} to be supported.");
             }
         }
+
+        [Given(@"I create a delayed button")]
+        public void GivenICreateADelayedButton()
+        {
+            string jsButton = Properties.Resources.ResourceManager.GetString("jsDelayedButton");
```

Comment:
Can we move the code that gets pulled in here into the step def instead of as a resource?